pravega/schema-registry
# Issue 61: Exposing schema registry as an ingress

Signed-off-by: SrishT <Srishti.Thakkar@dell.com>

**Change log description**  
Provides helm charts for deployment of schema registry on a Kubernetes cluster. Provides the users with the option of exposing schema registry as an ingress resource as well.

**Purpose of the change**  
Fixes #61 

**What the code does**  
Simplifies deployment of schema-registry for end users through the use of helm.

**How to verify it**  
The following command should successfully deploy schema-registry on your cluster
```
$ helm install <release-name> charts/schema-registry --set controllerUri='tcp://<controller-ip>:9090'
```
To install schema registry with ingress enabled using the following command
```
$ helm install sr charts/schema-registry --set ingress.enabled=true
```
This would create the ingress resource as we can see with the following command
```
$ kubectl get ingress
NAME                 HOSTS                         ADDRESS        PORTS   AGE
sr-schema-registry   schema-registry.pravega.com   10.243.39.70   80      23s
```
